### PR TITLE
Sandbox tool camera rotation and pitch control

### PR DESCRIPTION
**Pull request by rpartsey**
## Motivation and Context

Added 2 ways to control camera yaw and pitch: 
- WASD keys
- hold Q and move mouse

TODO: decide which one has better UX

## How Has This Been Tested

By launching Sandbox Tool locally.

## Types of changes

- **Refactoring**

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-lab/habitat/gui/gui_application.py [hunk 1]
```diff
@@ -107,8 +107,12 @@ def get_mouse_position(
 
     def mouse_move_event(self, event: Application.MouseMoveEvent) -> None:
         mouse_pos = self.get_mouse_position(event.position)
+        relative_mouse_position = self.get_mouse_position(
```
Comment (eundersander) [inline habitat-lab/habitat/gui/gui_application.py:110]: One thing I'm trying to do with this input wrapper is gracefully handle the case where we poll for input from the OS at a different rate from processing it by the sim.

Specifically, (1) what if we get multiple calls to mouse_move_event between calls to sim_update, or (2) what if we do multiple calls to sim_update between calls to mouse_move_event?

In the case of (1), we want the accumulated mouse movement available inside sim_update. For (2), we want the first sim_update to get the mouse movement but the second sim_update should get zero mouse movement.

TLDR make _relative_mouse_position work exactly like _mouse_scroll_offset.

**Review by eundersander — APPROVED** [review]
Small request on how we handle relative mouse movement. After that, LGTM!
